debian/pom.xml

```diff
@@ -77,6 +82,17 @@
                                         <prefix>/usr/share/corfu/lib</prefix>
                                     </mapper>
                                 </data>
+                                <!-- dependency of corfudb-tools is included -->
+                                <data>
+                                    <src>${session.executionRootDirectory}/corfudb-tools/target/corfudb-tools-${project.version}-shaded.jar</src>
+                                    <type>file</type>
+                                    <mapper>
+                                        <type>perm</type>
+                                        <prefix>/usr/share/corfu/lib</prefix>
+                                        <group>corfu</group>
+                                        <filemode>750</filemode>
```

Comment:
Perhaps other scripts can also be limited to the user and group - right now they seem pretty open?

Reply:
The other relaxed permissions are for the Clojure scripts. I'm not sure if we want to modify those are they are invoked directly for debugging and also to get out of nondeterministic layout states.